plugins/modules/nmcli.py

```diff
@@ -81,9 +83,17 @@
         type: str
         choices: [ datagram, connected ]
         version_added: 5.8.0
+    slave_type:
+        description:
+            - Type of the device of this slave's master connection (for example C(bond)).
+            - Mandatory if I(master) is defined.
```

Comment:
This looks like a breaking change. So far one could specify `master` *without* specifying `slave_type`. This must still be possible.

Reply:
It still possible. `slave_type` may be omitted when `type` is one of `bond-slave`, `bridge-slave` or `team-slave` - this types implies according `slave-type` on NM side. In this case module works as prior (checks not performing). `extra_options_validation` method implements related checks.

Reply:
Fixed in 3d82405